ckanext/ontario_theme/templates/internal/snippets/search_form.html

```diff
@@ -36,7 +91,7 @@
         missing attribute error.
       #}
       <span>
-        <span class="facet">with the keyword(s):</span>
+        <span class="facet">{{ _('with the keyword(s)') }}:</span>
```

Comment:
```suggestion
        {% trans %}<span class="facet">with the keyword(s):</span>{% endtrans %}
```

Reply:
I decided to make a separate punctuation element so that it can simply be re-used everywhere without being attached to actual texts:

```
# French punctuations
msgid ":"
msgstr " :"
```

e.g.:
```
<span class="facet">{{ _('with the keyword(s)') }}{{ _(':') }}</span>
```

Reply:
Actually given your earlier comment, I can add the nbsp as a span:
```
# French punctuations
msgid ":"
msgstr "<span>&nbsp;:</span>"
```